src/lib.rs

```diff
@@ -1,6 +1,8 @@
 #![allow(non_camel_case_types)]
 
-use std::os::raw::{c_char, c_int, c_void};
+use std::os::raw::{c_char, c_int, c_uchar, c_void};
+
+pub type size_t = usize;
```

Comment:
Alternatively could depend on the libc crate for this definition but that seems like overkill.

Reply:
Let's go ahead and use libc, with `default-features = false`. It doesn't add that much, and any decent size project will probably alreayd have it pulled in elsewhere anyway.

It would be strange if this were ever something other than `usize`, but better safe than sorry.